src/providers/camera/camera.provider.ts

```diff
@@ -8,35 +8,32 @@ export class CameraProvider {
   }
 
   getPictureFromCamera() {
-    return this.getImage(this.camera.PictureSourceType.CAMERA, true);
+    return this.getImage(this.camera.PictureSourceType.CAMERA);
   }
 
   getPictureFromPhotoLibrary() {
     return this.getImage(this.camera.PictureSourceType.PHOTOLIBRARY);
   }
 
   // This method takes optional parameters to make it more customizable
-  getImage(pictureSourceType, crop = true, quality = 50, allowEdit = false, saveToAlbum = true) {
+  getImage(pictureSourceType, quality = 50, saveToAlbum = true) {
+    //screen.msLockOrientation('landscape')
     let options = {
+      targetHeight: 600,
+      targetWidth: 600,
       quality: quality,
       destinationType: this.camera.DestinationType.DATA_URL,
       sourceType: pictureSourceType,
-      allowEdit: allowEdit,
```

Comment:
`allowEdit: allowEdit`
Aqui ele tá forçando para ser false caso o allowEdit não seja forçado para ser `true`